lib/db.js

```diff
@@ -654,10 +667,13 @@ module.exports = (
     }
     return P.all(promises)
       .spread((deleteRes, redisTokens) => {
-        const redisSessionTokens = redisTokens ? JSON.parse(redisTokens) : []
-        const tokenToDeleteIndex = redisSessionTokens.findIndex((tok) => tok.tokenId === sessionToken.id)
-        redisSessionTokens.splice(tokenToDeleteIndex, 1)
-        return this.redis.setAsync(uid, JSON.stringify(redisSessionTokens))
+        if (this.redis) {
```

Comment:
This is what https://github.com/mozilla/fxa-auth-server/pull/2001 did. The difficult part was having test cases to cover all endpoints that use redis and check they were still valid when enabled/disabled.

Either way I think I will close that PR in favor of this!